src/go/rpk/pkg/cli/transform/deploy.go

```diff
@@ -149,6 +153,7 @@ The --var flag can be repeated to specify multiple variables like so:
 	cmd.Flags().StringSliceVarP(&fc.outputTopics, "output-topic", "o", []string{}, "The output topic to write the transform results to (repeatable)")
 	cmd.Flags().StringVar(&fc.functionName, "name", "", "The name of the transform")
 	cmd.Flags().Var(&fc.env, "var", "Specify an environment variable in the form of KEY=VALUE")
+	cmd.Flags().StringVar(&fc.compression, "compression", "", "Output batch compression type")
```

Comment:
Shouldn't this be an explicit "none"?

Reply:
yeah fair point

Reply:
With the current flag-overrides-file model, wouldn't this preclude the ability to control the setting via YAML?

Reply:
Yeah I think so, if you ask me!